docs/source/reference/ha.rst

```diff
@@ -162,6 +162,14 @@ In a proper distributed setup it is recommended to setup Zookeeper or Redis to p
 distributed co-ordination layer. See :doc:`Policies </reference/policies>`. Using the default
 file-based co-ordination backend will not work as it would in a single box deployment.
 
+By default the ``st2actionrunner`` service runs 10 workers. This number can be increased by
+modifying the environment variable ``WORKERS``. To make this permanent, create or edit
+``/etc/sysconfig/st2actionrunner`` and add the following line:
```

Comment:
It's not :100: accurate. The number of WORKERS are determined depending on number of CPU cores machine has.
See 
https://github.com/StackStorm/st2-packages/blob/1fb2f7a60e5d88895a5dfed79c55edc68482dc43/packages/st2/debian/st2.st2actionrunner.upstart#L30-L34

Additionally, `/etc/sysconfig/st2actionrunner` is for EL. For Ubuntu the respective config file to place custom ENV variables for the service/unit is `/etc/default/st2actionrunner`.

See https://docs.stackstorm.com/packs.html#proxy-configuration-via-environment-variables as example.

Reply:
Updated accordingly

Reply:
Yes, looks good :+1: